addons/website_blog/static/tests/tours/blog_tags_with_date.js

```diff
@@ -0,0 +1,63 @@
+/** @odoo-module **/
+
+import wTourUtils from 'website.tour_utils';
+
+/**
+ * Makes sure that blog tags should not be removed on the additon of date filter
+ * and on the removal of date filter.
+ */
+wTourUtils.registerWebsitePreviewTour("blog_tags_with_date", {
+    test: true,
+    url: "/blog",
+    edition: true,
+}, [{
+        content: "Click on first blog",
+        trigger: "iframe article[name=blog_post] a",
+    }, {
+        content: "Click on sidebar option",
+        trigger: "we-customizeblock-options we-button[data-customize-website-views='website_blog.opt_blog_sidebar_show'] we-title",
```

Comment:
1. I would prefer to identify the any option by it's `data-name` attribute.
2. I know this trigger is working fine, but conventionally `we-title` is not clickable entity (currently it's clickable), you should target the button.